okdc-centos.sh

```diff
@@ -227,7 +244,7 @@ update_kubelet() {
 	Wants=flexv.service
 	After=flexv.service
 	[Service]
-	Environment="KUBELET_NETWORK_ARGS=--network-plugin=kubenet"
+	Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni"
```

Comment:
Have you tested this one on kubeadm 1.7? This used to lead to a deadlock while kubeadm init. Not sure if it's fixed. The previous method is to wait until kubeadm is done then remove this line in `patch_kubelet`.